lib/actions/pauseall.js

```diff
@@ -58,8 +58,27 @@ function doResumeAll(system) {
   return Promise.all(promises);
 }
 
+function anyPlayerOn(player, values) {
+  logger.debug("is any player on");
+
+  if (values[0] && values[0] > 0) {
+    setTimeout(function () {
+      doStateOn(player.system);
+    }, values[0] * 1000 * 60);
+    return Promise.resolve();
+  }
+
+  return doAnyPlayerOn(player.system);
+}
+
+function doAnyPlayerOn(system) {
+  const promises = system.zones.some(zone => zone.coordinator.state.playbackState === 'PLAYING');
+  return {promises};
```

Comment:
Nothing here is a promise, so not sure why you return an object with `{ promises: bool }`